tls/s2n_connection.h

```diff
@@ -331,6 +331,8 @@ struct s2n_connection {
      */
     bool send_in_use;
     bool recv_in_use;
+    
+    uint8_t tickets_to_send;
```

Comment:
It seems reasonable that a customer couldn't request more than 256 tickets initially, but should they be able to request more than 256 tickets over the lifetime of a connection?

Reply:
Increased limit to uint16_t.

Reply:
Increased ticket limit to uint16_t for now.